about.html

```diff
@@ -0,0 +1,127 @@
+        <li class="text-white "><i class="fa-brands fa-github"></i></li>
+        <li class="text-white">Logout</li>
+        <li class="text-white">Mypage</li>
+        <li class="text-white">India</li>
+      </ul>
+    </div>
+    <nav class="navbar navbar-expand-md  bg-white">
+      <div class="container toggles justify-content-md-evenly">
+        <a class="navbar-brand" href="index.html"><i class="fa-solid fa-won-sign fa-2xl"></i></a>
+        <button class=" btn burger d-md-none" type="button"><i class="fa-solid fa-bars"></i></button>
+        <button type="button" class="btn d-none close-btn"><i class="fa-solid fa-xmark "></i></button>
+        <div class="justify-content-evenly d-none d-md-flex" id="navbarNav">
+          <ul class="navbar-nav">
+            <li class="nav-item"><a class="nav-link " aria-current="page" href="about.html">About</a>
+            </li>
+            <li class="nav-item"><a class="nav-link" href="#Cards">Features</a>
+            </li>
+            <li class="nav-item"><a class="nav-link" href="#feature">Sources</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+  </header>
+  <div class="about">
+    <div class="intro ps-5">
+      <h1 class="fs-2 pb-3">Web Devlopment</h1>
+      <p class="border border-white pt-5 about-p">Web development is the work involved in developing a website for the
+        Internet or an intranet. Web development can range from developing a simple single static page of plain text to
+        complex web applications, electronic businesses, and social network services.
+      </p>
+        <h4 class="pt-4 intro-h4">For More Information go to </h4>
+        <a href="#" class="text-dark fs-4 fw-semibold"> W3school</a>
+    </div>
+    <hr class="hr-1">
+    <div class="webLogo text-center bg-white pb-5 ">
+      <h2 class="pt-5 fw-bolder">Web Development Roadmaps</h2>
+      <hr>
+      <h4 class="py-4 fw-bold">Front-End Roadmap</h4>
+      <p class="fw-semibold fs-4 p-lg-0 p-2">To become a web developer, start with the subjects below,
+        in the following order:</p>
+      <div class="d-flex flex-md-column align-items-center">
+        <a href="#" class="text-decoration-none web-btn ">HTML</a>
+        <a href="#" class="text-decoration-none web-btn">CSS</a>
+      <a href="#" class="text-decoration-none web-btn web-btn-1">JAAVASCRIPT</a>
+      </div>
+    </div>
+  </div>
+  <hr>
+  <div class="moreInfo pt-5 text-center bg-white pb-5" id="moreInfo">
+    <h3 class="fw-semibold pb-2">What's Next?</h3>
+    <hr>
+    <p class="fw-semibold p-3 "> Now you know how to use HTML, CSS, and JavaScript
+      to create, style, and make interactive web pages.
+      The next step is to publish your website,
+      so that the rest of the world can see your work.
+      There are tons of hosting services to choose from.
+      We have made one for you, for free:
+    </p>
+    <img src="images/web3school.png" alt="web3School" class="img">
```

Comment:
- [ ] [OPTIONAL] On Desktop, I kindly suggest that you make the images align horizontally just like it is on the design template.

**Template Design:**
<img width="556" alt="html-capstone" src="https://user-images.githubusercontent.com/52934128/181719329-9bb9e966-dd61-482e-bb72-306346a2c335.PNG">